src/test/java/SalesforceModules/SalesForceBillingUtil.java

```diff
@@ -337,6 +337,135 @@ public static void createNewContactOfContacts(String salutation, String name, St
+        }
+    }
+
+    /**
+     * Clicking new button on Cases
+     * @param logStep
+     * @throws InterruptedException
+     */
+    public static void clickOnNewBtnOfCases(String logStep) throws InterruptedException {
+        KeywordUtil.delay(2000);
+        KeywordUtil.scrollingToElementofAPage(BillingObject.newBtnOfCases,"Scrolling to the case");
+        KeywordUtil.waitForVisible(BillingObject.newBtnOfCases);
+        KeywordUtil.click(BillingObject.newBtnOfCases, logStep);
+    }
+
+    /**
+     * Create new case
+     * @param logStep
+     * @throws InterruptedException
+     */
+    public static void createNewCase(String origin, String logStep) throws InterruptedException {
+        clickOnNewBtnOfCases("Clicked on New button of cases");
+        KeywordUtil.delay(2000);
+        CasesUtil.setCaseOrigin(origin, "Case origin value selected");
+        CasesUtil.clickSaveButton("Click on save button");
+    }
+
+    /**
+     * Click on DropDown
+     * @param logStep
+     * @throws InterruptedException
+     */
+    public static void clickOnDropdown(String logStep) throws InterruptedException {
+        KeywordUtil.scrollingToElementofAPage(BillingObject.dropdownOfCases,"Scrolling to the dropdown");
+        KeywordUtil.waitForVisible(BillingObject.dropdownOfCases);
+        KeywordUtil.click(BillingObject.dropdownOfCases, logStep);
+        KeywordUtil.delay(3000);
+    }
+
+    /**
+     * Click on Edit Option
+     * @param logStep
+     * @throws InterruptedException
+     */
+    public static void clickOnEditOption(String logStep) throws InterruptedException {
+        KeywordUtil.waitForVisible(BillingObject.dropdownEditOption);
+        KeywordUtil.clickJS(BillingObject.dropdownEditOption, logStep);
+        KeywordUtil.delay(2000);
+    }
+
+    /**
+     * Click on Delete Option
+     * @param logStep
+     * @throws InterruptedException
+     */
+    public static void clickOnDeleteOption(String logStep) throws InterruptedException {
+        KeywordUtil.waitForVisible(BillingObject.dropdownDeleteOption);
+        KeywordUtil.clickJS(BillingObject.dropdownDeleteOption, logStep);
+        KeywordUtil.delay(2000);
+    }
+
+    /**
+     * Edit Create New case
+     * @param origin
+     * @throws InterruptedException
+     */
+    public static void editCreatedNewCase(String origin) throws InterruptedException {
+        clickOnDropdown("Click on dropdown");
+        clickOnEditOption("Click on edit option");
+        KeywordUtil.scrollingToElementofAPage(BillingObject.editOrigin,"Scroll till origin");
+        setCaseEditOrigin(origin, "Other value of case origin is selected");
+        CasesUtil.clickSaveButton("Click on save button");
+    }
+
+    /**
+     * Input the webCompany
+     * @param Company the Company
+     * @param logStep the logStep
+     * @throws InterruptedException
+     */
+    public static void webCompany(String Company, String logStep) throws InterruptedException {
```

Comment:
Argument name should start from small letter

Reply:
done